code/src/App.js

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
+import ThoughtWrapper from 'components/ThoughtWrapper';
+import Header from './components/Header';
```

Comment:
This was a great idea to keep the App.js clean just to import the two components :)